frontend2/src/utils/api/compete.ts

```diff
@@ -21,19 +22,13 @@ const API = new CompeteApi(DEFAULT_API_CONFIGURATION);
  */
 export const uploadSubmission = async (
   episodeId: string,
-  submission: {
-    file: File;
-    packageName: string;
-    description: string;
-  },
+  submission: SubmissionUploadRequest,
 ): Promise<void> => {
```

Comment:
1. We should have this return a Promise<Submission> which contains the submission received by our backend. 
2. we can then render the submission on our frontend in the submission history table. otherwise, the user will need to reload the page to see the new submission

The rerendering / storing of the updated submission is probably something that can be made easier with react-query, so I'm OK with waiting for that to be added before doing (2)